docs/user-guide/members/README.md

```diff
@@ -8,16 +8,62 @@ keywords: members, project participants
+| [Remove members](#remove-members)                     | How to remove members from a project.                        |
+| [Roles and permissions](#roles-and-permissions)       | How to manage roles and permissions for members.             |
+| [Groups](#groups)                                     | How to add members to a group and add groups to a project.   |
 
 <video src="https://openproject-docs.s3.eu-central-1.amazonaws.com/videos/OpenProject-Invite-and-Manage-Members.mp4" type="video/mp4" controls="" style="width:100%"></video>
 
+## Project members overview
+
+On the left side menu you will see **Members**. When selected, it will show a list of project members, project groups, as well as the users, with whom work packages from this project have been shared. You can **edit** or **delete** a user or a group by click the respective icon at the end of the line listing said user or group.
+
+![Project members overview in OpenProject](members-overview.png)
+
+Standard filters on the left side menu include the following: 
+
+- **All** - returns all members and groups of the project, as well as non-members, with whom a work package from this project has been shared
+
+- **Locked** - returns all locked users that are members of this project, as well as locked non-members, with whom a work package from this project has been shared
+
+- **Invited** - returns all users that have been invited, but have not yet registered
+
+- **Project roles** provides filters based on all the user roles that have been assigned to users in that specific project. 
+
+- **Work package shares** provides filters based on all the roles available for sharing work packages. They include
+- **All shares** - returns all users that a work package in this project has been shared with
+  
+- **View** - returns all users that can view, but not edit or comment on a work package that has been shared with them
+  
+- **Comment** - returns all users that are allowed to add comments to a work package that has been shared with them
+  
+- **Edit** - returns all users that are permitted to edit a work package that has been shared with them
+
+> Note: users, with whom work packages from a given project have been shared, can not be edited or deleted under **Members**. To edit or revoke their viewing rights please navigate to **Work packages**, select the **Shared with users** filter and adjust the privileges accordingly. [Read more](../work-packages/share-work-packages/#remove-sharing-privileges).
+
+- **Groups** lists all the groups that have been added to this project. 
+
+> Note: members that are part of a group will also be displayed as members individually. In that case you can only edit the roles assigned to the users, but not delete him or her. If you want to delete a user that is a member of a group (also added to this project) you will have to delete the entire group and add group members individually if needed.
```

Comment:
I think there is an easier path to edit or delete shared work packages of a single member. The admin could click on the X work package(s) in the column "Shared" in the "Members" page to access an already filtered work package list of all work packages shared with that user.

Reply:
I think this refers to ta different place in text, but I understand what you mean. Thanks.

Reply:
Sorry, it refers to the other Note section